js/main.js

```diff
@@ -197,55 +242,59 @@ $(window).bind('keydown', function(e) {
     Backbone.history.loadUrl();
   }
 
-  if (ctrl) {
-    switch (char) {
-      case config.keyShortcuts.undo:
-        //run undo programmatically to avoid crash
-        e.preventDefault();
-        document.execCommand('undo');
-        break;
-      case config.keyShortcuts.discover:
-        route = 'home';
-        break;
-      case config.keyShortcuts.myPage:
-        route = 'userPage';
-        break;
-      case config.keyShortcuts.customizePage:
-        route = 'userPage/' + user.get('guid') + '/customize';
-        break;
-      case config.keyShortcuts.create:
-        route = 'userPage/' + user.get('guid') + '/listingNew';
-        break;
-      case config.keyShortcuts.purchases:
-        route = 'transactions/purchases';
-        break;
-      case config.keyShortcuts.sales:
-        route = 'transactions/sales';
-        break;
-      case config.keyShortcuts.cases:
-        route = 'transactions/cases';
-        break;
-      case config.keyShortcuts.settings:
-        route = 'settings';
-        break;
-      case config.keyShortcuts.addressBar:
-        // Select all text in address bar
-        $('.js-navAddressBar').select();
-        break;
-      case config.keyShortcuts.save:
-        window.obEventBus.trigger('saveCurrentForm');
-        break;
-      case config.keyShortcuts.refresh:
-        Backbone.history.loadUrl();
-        break;
-    }
-
-    if (route !== null) {
+  switch (char) {
```

Comment:
Is there where @drwasho was saying the shortcuts no longer check for the control key? Why did you take it out?